src/cookbook/grant-roles.md

```diff
@@ -11,4 +11,16 @@ head:
 
 # How to Grant a Role
 
-TODO
\ No newline at end of file
+Roles are granted to accounts:
+
+```rust
+fn grant_role(
+    iroha: &Client,
```

Comment:
`client: &Client` may be an option too, though it isn't critical at all. Consistency it primary.

Reply:
Initially I did just that, but then went for this format to make it less redundant. I can comb through all the examples at some point in the future to enforce a style that we agree on.

Reply:
I find `iroha` more clear than `client`. Maybe also consider renaming `Client` to `Iroha`?

Reply:
`Iroha` is currently taken by the peer process, but they are not supposed to appear together, so maybe it's ok